client/layout/controller.js

```diff
@@ -213,52 +205,22 @@ export class Controller extends Component {
 	}
 }
 
-/**
- * Update an anchor's link in sidebar to include persisted queries. Leave excluded screens
- * as is.
- *
- * @param {HTMLElement} item - Sidebar anchor link.
- * @param {Object} nextQuery - A query object to be added to updated hrefs.
- * @param {Array} excludedScreens - wc-admin screens to avoid updating.
- */
-export function updateLinkHref( item, nextQuery, excludedScreens ) {
-	if ( isWCAdmin( item.href ) ) {
-		const search = last( item.href.split( '?' ) );
-		const query = parse( search );
-		const path = query.path || 'homescreen';
-		const screen = path.replace( '/analytics', '' ).replace( '/', '' );
-
-		const isExcludedScreen = excludedScreens.includes( screen );
-
-		const href =
-			'admin.php?' +
-			stringify(
-				Object.assign( query, isExcludedScreen ? {} : nextQuery )
-			);
-
-		// Replace the href so you can see the url on hover.
-		item.href = href;
+// Add click handlers to wc-admin links in wp-admin menu to suppress page refresh when possible
+const patchMenuLinks = function () {
```

Comment:
I renamed this from `window.wpNavMenuUrlUpdate` but I'm worried this could result in a breaking change somewhere. A quick search on github shows that hook being called, but as far as I can tell it's only in forks of our codebase? (specifically where we were zeroing these out in settings).